akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/Running.scala

```diff
@@ -36,6 +36,7 @@ import akka.persistence.query.{ EventEnvelope, PersistenceQuery }
 import akka.persistence.query.scaladsl.EventsByPersistenceIdQuery
 import akka.persistence.typed.ReplicaId
 import akka.persistence.typed.ReplicationId
+import akka.persistence.typed.internal.BehaviorSetup.SnapshotWithoutRetention
```

Comment:
Curious why an import needed to be added here when the only other change is a delete, accidental?

Reply:
Yeah, just didn't check CI after it ran. Also, because I develop on Windows machine, I have to disable discipline checks because of their issues with Windows, maybe they would have caught it. I'll fix this up in the evening.